partner_communication_switzerland/models/contracts.py

```diff
@@ -445,14 +448,11 @@ def contract_waiting_mandate(self):
         new_spons._new_dossier()
         new_spons.filtered(
             lambda s: s.correspondent_id.email and s.sds_state == 'draft' and
-            s.partner_id.ref != '1502623' and not
-            s.welcome_active_letter_sent).write({
-                'sds_state': 'waiting_welcome', 'sds_state_date':
+                      s.partner_id.ref != '1502623' and not
+                      s.welcome_active_letter_sent).write({
+            'sds_state': 'waiting_welcome', 'sds_state_date':
                 fields.Date.today()})
-        csp = self.filtered(
-            lambda s: '6014' in s.mapped(
-                'contract_line_ids.product_id.property_account_income_id.code')
-        )
+        csp = self.filtered(lambda s: 'CSP' in s.name)
```

Comment:
This is a step back, it was on purpose changed to the above version. Please revert this change.